app/src/main/java/com/example/lemonade/MainActivity.kt

```diff
@@ -90,11 +90,11 @@ fun LemonadeApp() {
                 )
             )
         }
-    ) { innerPadding ->
+    ) {  innerPadding ->
         Surface(
             modifier = Modifier
                 .fillMaxSize()
-                .padding(innerPadding)
+//                .padding(innerPadding)
```

Comment:
Is it ideal to not use `surface` padding and suppress the lint warning `UnusedMaterial3ScaffoldPaddingParameter`?

Reply:
You should not ignore the `innerPadding`. It contains the system bars as well as the app bars.

Reply:
Thank you